@xen-orchestra/rest-api/src/backup-repositories/backup-repositories.controller.mts

```diff
@@ -0,0 +1,56 @@
+import { Example, Get, Path, Query, Request, Response, Route, Security, Tags } from 'tsoa'
+import { provide } from 'inversify-binding-decorators'
+import { Request as ExRequest } from 'express'
+import type { XoBackupRepository } from '@vates/types'
+
+import { notFoundResp, unauthorizedResp, type Unbrand } from '../open-api/common/response.common.mjs'
+import {
+  backupRepositoryIds,
+  partialBackupRepositories,
+  backupRepository,
+} from '../open-api/oa-examples/backup-repository.oa-example.mjs'
+import type { SendObjects } from '../helpers/helper.type.mjs'
+import { XoController } from '../abstract-classes/xo-controller.mjs'
+
+@Route('backup-repositories')
+@Security('*')
+@Response(unauthorizedResp.status, unauthorizedResp.description)
+@Tags('backup-repositories')
+@provide(BackupRepositoryController)
+export class BackupRepositoryController extends XoController<XoBackupRepository> {
+  // --- abstract methods
+  getAllCollectionObjects(): Promise<XoBackupRepository[]> {
+    return this.restApi.xoApp.getAllRemotes()
+  }
+  getCollectionObject(id: XoBackupRepository['id']): Promise<XoBackupRepository> {
+    return this.restApi.xoApp.getRemote(id)
+  }
+
+  /**
+   * @example fields "id,name"
+   * @example filter "enabled?"
+   * @example limit 42
```

Comment:
Try to always add the `fields` properties you are filtering on (just to be more user-friendly)
```suggestion
   * @example fields "id,name,enabled"
   * @example filter "enabled?"
   * @example limit 42
```